src/textual/color.py

```diff
@@ -54,6 +54,8 @@ class Lab(NamedTuple):
 
 RE_COLOR = re.compile(
     r"""^
+\#([0-9a-fA-F]{3})$|
```

Comment:
Shouldn't we start naming these capture groups, in order to make the parsing a bit clearer and rely less on the unpacking of a tuple with 6 "position-only" items?

Reply:
I suspect it's a little faster to parse with positional arguments. Unpacking in to locals is very quick compared to a number of dict lookups.

Reply:
I agree it would probably be overall beneficial in some cases we have. In this case though the capture group is so short I actually think the regex is clearer without it.